### PR TITLE
Template override link

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
Changes the badge with the count of files to a link directly to the correct tab of the template manager

### Expected result AFTER applying this Pull Request

image

### CODE REVIEW

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on b601004c5ce76707f00a577dd7d9c30150adefc0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38448">issues.joomla.org/tracker/joomla-cms/38448</a>.</sub>

File: administrator/components/com_templates/tmpl/templates/default.php [hunk 1]
```diff
@@ -118,7 +118,9 @@
                                 <?php if ($this->pluginState) : ?>
                                     <td class="d-none d-md-table-cell text-center">
                                         <?php if (!empty($item->updated)) : ?>
-                                            <span class="badge bg-warning text-dark"><?php echo Text::plural('COM_TEMPLATES_N_CONFLICT', $item->updated); ?></span>
+                                            <a href="<?php echo Route::_('index.php?option=com_templates&view=template&id=' . (int) $item->extension_id . '#files');?>">
```
Comment (Quy) [inline administrator/components/com_templates/tmpl/templates/default.php]:
```suggestion
                                            <a href="<?php echo Route::_('index.php?option=com_templates&view=template&id=' . (int) $item->extension_id . '#files'); ?>">
```

**Comment by Abernyte-Git** [comment]
I have tested this item :white_check_mark: successfully on b601004c5ce76707f00a577dd7d9c30150adefc0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38448">issues.joomla.org/tracker/joomla-cms/38448</a>.</sub>

File: administrator/components/com_templates/tmpl/templates/default.php (hunk 1 above)
Comment (richard67) [reply administrator/components/com_templates/tmpl/templates/default.php]: @brianteeman Could you check @Quy 's suggestion? Thanks in advance.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38448">issues.joomla.org/tracker/joomla-cms/38448</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Comment by brianteeman** [comment]
Please correct the milestone to 4.2.2

**Pull request merged**